pkgs/development/python-modules/mplhep/default.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, hist
+, matplotlib
+, mplhep-data
+, pytestCheckHook
+, pytest-mock
+, pytest-mpl
+, scipy
+, setuptools
+, setuptools-scm
+, uhi
+, uproot
+}:
+
+buildPythonPackage rec {
+  pname = "mplhep";
+  version = "0.3.28";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-/7nfjIdlYoouDOI1vXdr9BSml5gpE0gad7ONAUmOCiE=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = [
+    matplotlib
+    uhi
+    mplhep-data
+  ];
+
+  checkInputs = [
```

```suggestion
  nativeCheckInputs = [
```
We should use `nativeCheckInputs` here.
see, https://github.com/NixOS/nixpkgs/pull/206742